grpc/proto/v1/search_get.proto

```diff
@@ -49,6 +49,7 @@ message SearchRequest {
   optional Rerank rerank = 61;
 
   bool uses_123_api = 100 [deprecated = true];
+  bool uses_125_api = 101 [deprecated = true];
```

Comment:
this should not be deprecated?

Reply:
It's deprecated in the sense that it will be removed once that `1.25` is the minimum actively supported version. Or do you think it should remain (almost) forever now the gRPC API is fully supported?

Reply:
I was thinking about deprecating it when 1.25 is the minimum actively supported version and removing it a bit later?